package.json

```diff
@@ -71,5 +71,8 @@
         "eslint": "^7.32.0",
         "eslint-plugin-import": "^2.24.0",
         "typescript": "^3.7.0"
-    }
+    },
+    "extensionPack": [
+        "dan-c-underwood.arm"
```

Comment:
From what I understood from the vscode docs and source, the `extensionPack` field is only valid for extension packs.

As an example:
- here is a package.json for cpptools extension: https://github.com/microsoft/vscode-cpptools/blob/c61bd07bb574dc4eeec4df2d4dc273e89cb445d5/Extension/package.json
- and here is a package.json for cpptools extension pack: https://github.com/microsoft/vscode-cpptools/blob/c61bd07bb574dc4eeec4df2d4dc273e89cb445d5/ExtensionPack/package.json#L37

Even though they are in the same repo, they are packaged separately.

Also, there are some downsides in turning this extension itself into an extension pack: e.g. if it becomes disabled, all of the "child" extensions will be disabled as well. In this case, disabling vscode-disasexpl will lead to disabled syntax highlighting for arm assembly.

I tried to find some kind of "recommends" field for this kind of thing, but it looks like this thing does not exist — only recommendations for a workspace (https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions). 

So, maybe it's just worth mentioning this extension in a readme instead?

Reply:
Thanks!